tests/data/packages/BrokenEmitsUTF8/setup.py

```diff
@@ -9,9 +7,9 @@ class FakeError(Exception):
 
 if sys.argv[1] == 'install':
     if hasattr(sys.stdout, 'buffer'):
-        sys.stdout.buffer.write('\nThis package prints out UTF-8 stuff like:\n'.encode('utf-8'))
-        sys.stdout.buffer.write('* return type of ‘main’ is not ‘int’\n'.encode('utf-8'))
-        sys.stdout.buffer.write('* Björk Guðmundsdóttir [ˈpjœr̥k ˈkvʏðmʏntsˌtoʊhtɪr]'.encode('utf-8'))
+        sys.stdout.buffer.write(b'\nThis package prints out UTF-8 stuff like:\n')
+        sys.stdout.buffer.write('* return type of ‘main’ is not ‘int’\n'.encode())
+        sys.stdout.buffer.write('* Björk Guðmundsdóttir [ˈpjœr̥k ˈkvʏðmʏntsˌtoʊhtɪr]'.encode())
```

Comment:
Is it possible to skip some rules from pyupgrade? IIRC this is not a universally liked preference.

Reply:
Agreed, I'm against it - explicit is better than implicit, etc.

Reply:
There are flags to disable some of the rules, but not this one. I'll blacklist the file instead.

Reply:
Yes but if the rule is "encode() doesn't need utf-8 specifying" then it's the *rule* I object to, so blacklisting the file isn't going to help.

Reply:
I'll submit a PR to pyupgrade. The when it lands the blacklist can be removed.

Reply:
https://github.com/asottile/pyupgrade/pull/416

Reply:
My PR got closed with the comment:
> no thanks -- the existing options are for breakages only. this is an opinionated tool without preferences

What should we do now?

Reply:
One option would be to fork pyupgrade. Just delete the plugin that applies this particular transformation - should not be too hard to maintain.

Reply:
Another idea: add another pre-commit hook to transform `str.format()` back to explicit `str.format('utf8')`

Reply:
No, that'll unconditionally break the pre-commit runs on CI, since it'll detect changes as a failure.